### PR TITLE
allows zone.js to cleanup error stacktraces

### PR DESCRIPTION
**Pull request by aaomidi**
Before:
```
  ERR Item is disposed.: Error: Item is disposed.
    at Promise (file:///D:/azuredatastudio/out/vs/base/parts/tree/browser/treeModel.js:45:30)
    at new ZoneAwarePromise (d:\azuredatastudio\node_modules\zone.js\dist\zone-node.js:891:29)
    at Lock.run (file:///D:/azuredatastudio/out/vs/base/parts/tree/browser/treeModel.js:43:20)
    at event_1.Event.once (file:///D:/azuredatastudio/out/vs/base/parts/tree/browser/treeModel.js:38:37)
    at e (file:///D:/azuredatastudio/out/vs/base/common/event.js:37:37)
    at Emitter.fire (file:///D:/azuredatastudio/out/vs/base/common/event.js:463:38)
    at LockData.dispose (file:///D:/azuredatastudio/out/vs/base/parts/tree/browser/treeModel.js:19:33)
    at fn.then (file:///D:/azuredatastudio/out/vs/base/parts/tree/browser/treeModel.js:50:26)
    at ZoneDelegate.invoke (d:\azuredatastudio\node_modules\zone.js\dist\zone-node.js:388:26)
    at Zone.run (d:\azuredatastudio\node_modules\zone.js\dist\zone-node.js:138:43)
    at d:\azuredatastudio\node_modules\zone.js\dist\zone-node.js:872:34
    at ZoneDelegate.invokeTask (d:\azuredatastudio\node_modules\zone.js\dist\zone-node.js:421:31)
    at Zone.runTask (d:\azuredatastudio\node_modules\zone.js\dist\zone-node.js:188:47)
    at drainMicroTaskQueue (d:\azuredatastudio\node_modules\zone.js\dist\zone-node.js:595:35)
    at ZoneTask.invokeTask (d:\azuredatastudio\node_modules\zone.js\dist\zone-node.js:500:21)
    at Socket.ZoneTask.invoke (d:\azuredatastudio\node_modules\zone.js\dist\zone-node.js:485:48)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at Pipe.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
```
After:
```
  ERR Item is disposed.: Error: Item is disposed.
    at Promise (file:///D:/azuredatastudio/out/vs/base/parts/tree/browser/treeModel.js:45:30) [<root>]
    at new ZoneAwarePromise (d:\azuredatastudio\node_modules\zone.js\dist\zone.js:891:29) [<root>]
    at Lock.run (file:///D:/azuredatastudio/out/vs/base/parts/tree/browser/treeModel.js:43:20) [<root>]
    at event_1.Event.once (file:///D:/azuredatastudio/out/vs/base/parts/tree/browser/treeModel.js:38:37) [<root>]
    at e (file:///D:/azuredatastudio/out/vs/base/common/event.js:37:37) [<root>]
    at Emitter.fire (file:///D:/azuredatastudio/out/vs/base/common/event.js:463:38) [<root>]
    at LockData.dispose (file:///D:/azuredatastudio/out/vs/base/parts/tree/browser/treeModel.js:19:33) [<root>]
    at fn.then (file:///D:/azuredatastudio/out/vs/base/parts/tree/browser/treeModel.js:50:26) [<root>]
    at d:\azuredatastudio\node_modules\zone.js\dist\zone.js:872:34 [<root>]
    at drainMicroTaskQueue (d:\azuredatastudio\node_modules\zone.js\dist\zone.js:595:35) [<root>]
    at process._tickCallback (internal/process/next_tick.js:68:7) [<root>]
```

### CODE REVIEW

**Comment by anthonydresser** [comment]
I had this when I was implementing another feature and was having issues with zonejs
https://github.com/microsoft/azuredatastudio/blob/feat/highPerfTable/src/vs/code/electron-browser/workbench/workbench.js#L34

**Comment by aaomidi** [comment]
Looks like this is just scrubbing the current errors so we shouldn't get anything weird. 

Let's revert it if it fails?

File: src/sql/setup.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-define(["require", "exports"], function (require) {
+define(["require", "vs/base/common/errors", "exports"], function (require, errors) {
```
Comment (anthonydresser) [inline src/sql/setup.js]: I think imports have to be after require and exports. the loader expects require and exports to be the first two.

Comment (aaomidi) [reply src/sql/setup.js]: Ah thanks!

**Comment by kburtram** [comment]
@aaomidi do we need this?  The additional frames in the stack trace don't really bother me, and I'm not sure really what these code changes are doing.

**Comment by aaomidi** [comment]
Actually let me run this again. It's not just the additional stack traces rather the stack traces show where the code is being ran from rather than just showing zone.js.

**Comment by aaomidi** [comment]
Changes from: 
```
[10000:1003/131011.432:INFO:CONSOLE(2280)] "%c[Extension Host] %cdebugger listening on port 5871", source: d:\azuredatastudio\node_modules\zone.js\dist\zone-node.js (2280)
[10000:1003/131011.574:INFO:CONSOLE(2280)] "%c[Extension Host] %cdebugger inspector at chrome-devtools://devtools/bundled/inspector.html?experiments=true&v8only=true&ws=127.0.0.1:5871/b7aa5028-130f-4f39-8c7b-83809a17bd59", source: d:\azuredatastudio\node_modules\zone.js\dist\zone-node.js (2280)
```

To:
```
[48628:1003/131307.894:INFO:CONSOLE(214)] "%c[Extension Host] %cProvided debugging port 5870 is not free, using 5871 instead.", source: file:///D:/azuredatastudio/out/vs/workbench/services/extensions/electron-browser/extensionHost.js (214)
[48628:1003/131307.895:INFO:CONSOLE(220)] "%c[Extension Host] %cdebugger listening on port 5871", source: file:///D:/azuredatastudio/out/vs/workbench/services/extensions/electron-browser/extensionHost.js (220)

```

**Comment by aaomidi** [comment]
Yeah if we see it's affecting us negatively I'll remove it. It's nice to have for debugging test runs too :)

**Pull request merged**